<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleDlg</class>
 <widget class="QDialog" name="StyleDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="Lines_rbt">
        <property name="text">
         <string>Lines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Sticks_rbt">
        <property name="text">
         <string>Sticks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Steps_rbt">
        <property name="text">
         <string>Steps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Dots_rbt">
        <property name="text">
         <string>Dots</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ErrorBars_rbt">
        <property name="text">
         <string>Error Bars</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>213</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButtonStyle">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButtonStyle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
